CMakeLists.txt

```diff
@@ -158,17 +175,76 @@ macro(find_graphics)
   endif()
 endmacro(find_graphics)
 
-if (NOT MSVC)
```

Comment:
It just got moved down, into the top of the `else()` clause (after `if(MSVC)` on line 244) at lines 283-286.
I was trying to be very cautious of editing any "global" `CMAKE_*` settings, at least until after including the mimalloc submodule.

Reply:
I think it needs to be here, otherwise CGAL will not pick up the correct dependencies.

Reply:
OK, so it should be good as long as it comes before the CGAL finding block?  I'm not sure why that cgal block got moved up, I think that was just some confusion on my part while resolving a merge conflict.  I'll put that back where it was.

Reply:
Thx. This may actually have been some refactoring I did after you initially opened this branch :)
That region of the CMakeFile isn't particularly pretty, but it's so scary to touch it, unless all possible builds are available in the CI..